Tests/ContentKeyCrudTests.cs

```diff
@@ -83,6 +84,11 @@ public void AddContentKey_WithVariousValidData_Succeeds()
 				Value = new byte[Constants.ValidContentKeyLengthsInBytes.First()],
 				CommonEncryptionScheme = "cbcs"
 			})));
+			Assert.Null(Record.Exception(() => document.ContentKeys.Add(new ContentKey
```

Comment:
This checks that no exception occurred?
If so, is there not a Assert.DoesNotThrow() or similar function that could be used instead?

Reply:
I kept the same pattern, otherwise styles may mix?

Reply:
Yes, this seems to be the standard XUnit way, which is different from NUnit.